bluechi.spec.in

```diff
@@ -167,27 +171,21 @@ SELinux policy associated with the bluechi and bluechi-agent daemons
 %post selinux
 # Remove hirte policy
 if [ $1 -eq 1 ]; then
-   semanage port -N -d -p udp 842 2>/dev/null || true
-   semanage port -N -d -p tcp 842 2>/dev/null || true
    semodule -N -X 200 -r hirte 2>/dev/null || true
 fi
 %selinux_modules_install -s %{selinuxtype} %{_datadir}/selinux/packages/bluechi.pp.bz2
 restorecon -R %{_bindir}/bluechi* &> /dev/null || :
-semanage port -a -t bluechi_port_t -p udp 842 2>/dev/null || semanage port -m -t bluechi_port_t -p udp 842
-semanage port -a -t bluechi_port_t -p tcp 842 2>/dev/null || semanage port -m -t bluechi_port_t -p tcp 842
+%if %{with_selinux_port_any}
+   setsebool -P bluechi_controller_port_bind_any 1
```

Comment:
combine these lines together, to save install time.

  setsebool -P bluechi_controller_port_bind_any=1 bluechi_agent_port_connect_any=1

Reply:
Good point, changed it.